test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NetworkCallCountTest.cs

```diff
@@ -388,7 +388,7 @@ public void NetworkCallCount_RestoreLargePackagesConfigWithMultipleSourcesMainly
             }
         }
 
-        [Fact]
+        [PlatformFact(Platform.Windows, SkipMono = true)]
```

Comment:
`PlatformFact` as Windows would only allow it to run on windows. so does `skipMono` really makes sense here? And how about Linux?